ETMovies/ETMovies/Service/DataService.cs

```diff
@@ -0,0 +1,361 @@
+﻿using ETMovies.DatabaseContext;
```

Comment:
its nice that you use regions for each entity, but I would recommend to modularize it and create separate files for each entity